syntax = "proto3";

import "paxos.proto";
import "client.proto";

package proto.message;

message MessageWrapper {
  oneof message_wrapper {
    proto.client.ClientMessage client_message = 1;
    proto.paxos.PaxosMessage paxos_message = 2;
  }
}
